TASagentTwitchBot.Core/Config/BotConfiguration.cs

```diff
@@ -1,6 +1,9 @@
 ﻿using System;
 using System.Text.Json;
 using System.IO;
+using TASagentTwitchBot.Core.Core;
+using System.Text.Json;
+using System.IO;
```

Comment:
I like to keep using statements sorted approximately by scope.  System first, then external modules, then internal modules, then namespaces within the project.  But it also looks like the System.IO and System.Text.Json that were added here were duplicated from lines 2 and 3, so they should just be removed